pkg/sql/logictest/testdata/logic_test/show_external_connections

```diff
@@ -0,0 +1,70 @@
+# Here are tests for basic output of SHOW EXTERNAL CONNECTIONS
+# and various privileges scenarios. Tests for URI redaction for cloud providers
+# are located at pkg/ccl/cloudccl/externalconn/testdata
```

Comment:
Highlighting this comment. Creating external connection wasn't possible here so I placed the tests for URI redaction at the above mentioned location.